packages/web/src/lib/web-mode.ts

```diff
@@ -31,7 +31,8 @@ export class WebMode {
 	 * @param txId Transaction id
```

Comment:
could you please add `waitRounds` here as well?

Reply:
Oh! I forgot... Thanks <3